src/test/functionsDeployHelper.spec.ts

```diff
@@ -284,4 +295,453 @@ describe("functionsDeployHelper", () => {
       expect(promptStub).not.to.have.been.called;
     });
   });
+
+  describe("createDeploymentPlan", () => {
+    it("should put new functions into functionsToCreate", () => {
+      const regionMap: helper.RegionMap = {
+        "us-east1": [
+          {
```

Comment:
Might be easier to read these tests if you had a helper to generate the basic schema here.